src/Numerics/Mesh/Filters.jl

```diff
@@ -140,28 +140,37 @@ value of the filter function.
 struct ExponentialFilter <: AbstractSpectralFilter
     "filter matrix"
     filter::Any
+    "direction in which you're applying the filter"
+    direction::Direction
```

Comment:
So we opted to add `direction` as an attribute to the filter. The reason for this is that the filtering matrix you want to use is different depending on what direction you're applying it.

Reply:
An alternative approach might be to store in the filter object a tuple of filter matrices, one for each direction. This is similar to how the grid now contains tuple of differentiation matrices. Think it would help with application of a filter in `EveryDirection` too.

Reply:
@mwarusz Good idea! I like this idea better